app/src/main/java/org/eyeseetea/malariacare/layout/adapters/survey/DynamicTabAdapter.java

```diff
@@ -493,28 +493,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 break;
             case Constants.REMINDER:
             case Constants.WARNING:
-                TableRow.LayoutParams params = new TableRow.LayoutParams(TableRow.LayoutParams.MATCH_PARENT, TableRow.LayoutParams.MATCH_PARENT,0.5f);
 
-                int paddingSize= (int) PreferencesState.getInstance().getContext().getResources().getDimension(R.dimen.question_padding);
-
-                tableRow=(TableRow)lInflater.inflate(R.layout.dynamic_tab_row_singleitem, tableLayout, false);
-                tableRow.setLayoutParams(params);
+                tableRow=(TableRow)lInflater.inflate(R.layout.dynamic_tab_row_question_text, tableLayout, false);
```

Comment:
@idelcano I am going to merge it (it works as a charm!) but we are initialising tableRow twice here. This code can be refactor a little bit but obviously this is not the right moment!